src/dawn/IIR/ControlFlowDescriptor.h

```diff
@@ -37,6 +37,7 @@ class ControlFlowDescriptor {
 
   /// @brief Get the stencil description AST
   const StmtContainer& getStatements() const { return controlFlowStatements_; }
+  StmtContainer& getStatements() { return controlFlowStatements_; }
```

Comment:
Are they needed non-const?

Reply:
Yes. Before it worked because `ASTStmt` in `Statement` was basically non-const: https://github.com/MeteoSwiss-APN/dawn/blob/a45f64da5c43d64d0edffdae53974598f7d38497/src/dawn/SIR/Statement.h#L33
Now we need the non-const getter because of cases such as this:
https://github.com/MeteoSwiss-APN/dawn/blob/1f756e85de58d1e3e4114720fc47b6c79c329ff9/src/dawn/Optimizer/OptimizerContext.cpp#L238